pkg/storage/s3/s3.go

```diff
@@ -369,7 +369,11 @@ func (d *driver) CreateStorage(cr *imageregistryv1.Config) error {
 				Bucket: aws.String(d.Config.Bucket),
 			})
 			if err != nil {
-				if aerr, ok := err.(awserr.Error); ok {
+				if aerr, ok := err.(awserr.RequestFailure); ok {
+					err := fmt.Errorf("error requesting endpoint: %s(%d)", aerr.Code(), aerr.StatusCode())
```

Comment:
I would rather not fix the bug than lose the error message.

Reply:
Let's log it?